src/main/java/com/danielpsf/labs/field/statistic/metric/domain/MetricRequest.java

```diff
@@ -1,14 +1,19 @@
 package com.danielpsf.labs.field.statistic.metric.domain;
 
+import lombok.AllArgsConstructor;
 import lombok.Builder;
 import lombok.Getter;
 
 import java.util.Date;
 
 @Getter
 @Builder
+@AllArgsConstructor
 public class MetricRequest {
 
     private Double vegetation;
     private Date occurrenceAt;
+
+    public MetricRequest() {
```

Comment:
This can be considered a bad practice, but it was necessary due to a `jackson` exception I was getting on the integration test.

**Reference**:
- Item 3 (https://www.baeldung.com/jackson-exception)